src/openms/source/CHEMISTRY/EnzymesDB.cpp

```diff
@@ -274,6 +282,19 @@ namespace OpenMS
     }
   }
 
+  void EnzymesDB::getAllCOMETNames(vector<String>& all_names) const
+  {
+    all_names.clear();
+    all_names.push_back("unspecific cleavage");
```

Comment:
I copied this way from the getAllOMSSANames function:
It seemed to me that since the CometID is an integer, empty entries will be zero, so we ignore them. However, as the CometID  for unspecific cleavage is zero as well, we need to add it explicitly to the list before the loop..